runtime/src/bank.rs

```diff
@@ -4053,13 +4053,18 @@ impl Bank {
         error_counters: &mut TransactionErrorMetrics,
     ) -> Vec<TransactionCheckResult> {
         let hash_queue = self.blockhash_queue.read().unwrap();
+        let enable_durable_nonce = self
+            .feature_set
+            .is_active(&feature_set::separate_nonce_from_blockhash::id());
```

Comment:
But after this feature activation, if a transaction is executed is once as_either_ normal or durable transaction, then its nonce will not be equal to a blockhash. So it cannot be re-exectued, no?

Reply:
It can still be re-executed because the durable tx would be processed as a normal tx and if it failed, the advance nonce ix state changes would be reverted.

Reply:
could continue storing the raw blockhash, but switch domains on compare

Reply:
How would a 2nd feature activation help though?

Reply:
> I don't think we use the fee calculator value in the nonce data anymore and we could use that data to flag nonce hashes that use the new scheme and only allow processing those

Never mind, looks like it's still used. How about using a new version for the nonce state so that we can enforce that only the new type of nonce hashes can be accepted? Right now we have no way to distinguish.